cmd/chronicle/main.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/gookit/color"
+
+	"github.com/anchore/chronicle/cmd/chronicle/cli"
+)
+
+func main() {
```

Comment:
we should also be consistent with where build flags are, which in quill have been migrated to main for simplicity, and also to communicate that they are a concern for only the main package of this application https://github.com/anchore/quill/blob/17d942431c4b1537df0c4f6a02f8231b322d9d51/cmd/quill/main.go#L19-L23

(note: this also requires an update to the goreleaser config)